conversations/v3/common.yaml

```diff
@@ -439,6 +437,18 @@ components:
           description: Point in time when status has been entered
     MessageDeliveryEvents:
       type: object
+      required:
+        - messageId
+        - events
+      properties:
+        messageId:
+          type: string
+          format: uuid
+          description: Global unique Message Id reference
+        events:
+          type: array
+          items:
+            type: object
```

Comment:
Here, I've tried to document the schema. I've documented only what I know about it. Martin guesses that we can copy `MessageHistory` from v2 (https://github.com/tyntec/api-collection/blob/master/conversations/v2/common.yaml#L515). Is that right?

Reply:
Thank you.

I've send you a note on Slack with additional information

Reply:
Updated according to the note: https://github.com/tyntec/api-collection/pull/102/commits/a9198b9f0cdba87690367b66b26c1da5a2a2a384